packages/viz/src/modules/print/utils/constants.js

```diff
@@ -22,3 +22,15 @@ export const Images = {
   siteChangeTubingImage: null,
   siteChangeReservoirDiabeloopImage: null,
 };
+export const Fonts = {
+  default: {
+    regularName: "Helvetica",
+    boldName: "Helvetica-Bold",
+  },
+  ja: {
+    regular: null,
+    regularName: null,
```

Comment:
Why not set "NotoSerifJP-Regular" directly here? Same for bold name with "NotoSerifJP-Bold"?